### DISCUSSION TITLE
Set skipTargetFind to some object only?

### DISCUSSION THREAD

**Discussion by Tennyleaz** [fabricjs/fabric.js#8067] [Q&A] [answered]
Hi,
I would like to have all of my canvas object non-selectable, besides some only. I'm trying to use something like:

```javascript
// disable target find completly on other objects
canvas.skipTargetFind = true
// allow Rect objects to be selectable
fabric.Rect.prototype.selectable = true
```

But it seems `skipTargetFind` will suppress `selectable`. Any way to apply target find to some kind of objects only?

**Comment by ShaMan123** [comment, marked as answer]
mark all the rest as not `selectable`

**Reply by Tennyleaz** [reply]
But they could still interact with mouse, showing a move cursor when mouse hover?

**Reply by ShaMan123** [reply]
`evented=false`
